_components/Navigation.css

```diff
@@ -1,6 +1,6 @@
 .nav {
   height: 100%;
-  min-width: 280px;
+  min-width: 16rem;
```

Comment:
I think it's better to set width/height in `rem` wherever possible because if a user prefers a large font size, we should adjust the spacing to accommodate.